lib/oneboxer.rb

```diff
@@ -458,20 +458,20 @@ def self.external_onebox(url, available_strategies = nil)
       user_agent_override = SiteSetting.cache_onebox_user_agent if Oneboxer.cache_response_body?(url) && SiteSetting.cache_onebox_user_agent.present?
       onebox_options[:user_agent] = user_agent_override if user_agent_override
 
-      r = Onebox.preview(uri.to_s, onebox_options)
+      preview_result = Onebox.preview(uri.to_s, onebox_options)
```

Comment:
Unrelated to this fix but in the same sphere, I just really dislike these one letter variables used for anything more than an incrementer.